src/sql/strong-oracle/create.sql

```diff
@@ -3,255 +3,211 @@ CREATE SEQUENCE L_SEQ_ACCOUNT_PK
-  "DESCRIPTION"           CHARACTER VARYING(255) NULL
+  "REJECTION_REASON_ID" INTEGER NOT NULL,
+  "NAME" VARCHAR2(10) NOT NULL,
+  "DESCRIPTION" VARCHAR2(255) NULL
 );
-/
 
 CREATE INDEX "L_XPK_LU_TRANSFERS_REJECTION_R" ON "L_LU_REJECTION_REASON"
   ("REJECTION_REASON_ID" ASC);
-/
-
 ALTER TABLE "L_LU_REJECTION_REASON" ADD CONSTRAINT
   "L_PK_LU_TRANSFERS_REJECTION_RE"
   PRIMARY KEY ("REJECTION_REASON_ID");
-/
-
 CREATE INDEX "L_XAK_LU_TRANSFERS_REJECTION_R" ON "L_LU_REJECTION_REASON"
   ("NAME" ASC);
-/
-
 ALTER TABLE "L_LU_REJECTION_REASON" ADD CONSTRAINT
   "L_AK_LU_TRANSFERS_REJECTION_RE" UNIQUE
   ("NAME");
-/
+
 
 CREATE TABLE "L_LU_TRANSFER_STATUS" (
-  "STATUS_ID"             INTEGER NOT NULL,
-  "NAME"                  CHARACTER VARYING(20) NOT NULL,
-  "DESCRIPTION"           CHARACTER VARYING(255) NULL
+  "STATUS_ID" INTEGER NOT NULL,
+  "NAME" VARCHAR2(20) NOT NULL,
+  "DESCRIPTION" VARCHAR2(255) NULL
 );
-/
 
 CREATE INDEX "L_XPK_LU_TRANSFER_STATUS" ON "L_LU_TRANSFER_STATUS"
   ("STATUS_ID" ASC);
-/
-
 ALTER TABLE "L_LU_TRANSFER_STATUS" ADD CONSTRAINT "L_PK_LU_TRANSFERS_STATUS"
   PRIMARY KEY ("STATUS_ID");
-/
-
 CREATE INDEX "L_XAK_LU_TRANSFER_STATUS" ON "L_LU_TRANSFER_STATUS"
   ("NAME" ASC);
-/
-
 ALTER TABLE "L_LU_TRANSFER_STATUS" ADD CONSTRAINT "L_AK_LU_TRANSFER_STATUS"
   UNIQUE ("NAME");
-/
+
 
 CREATE TABLE "L_TRANSFERS"
 (
-  "TRANSFER_ID"            INTEGER NOT NULL,
-  "TRANSFER_UUID"          VARCHAR2(36) NOT NULL,
-  "LEDGER"                 VARCHAR2(1024) NULL,
-  "STATUS_ID"              INTEGER NOT NULL,
-  "REJECTION_REASON_ID"    INTEGER NULL,
-  "ADDITIONAL_INFO"        VARCHAR2(4000) NULL,
-  "EXECUTION_CONDITION"    VARCHAR2(4000) NULL,
+  "TRANSFER_ID" INTEGER NOT NULL,
+  "TRANSFER_UUID" VARCHAR2(36) NOT NULL,
+  "LEDGER" VARCHAR2(1024) NOT NULL,
+  "STATUS_ID" INTEGER NOT NULL,
+  "REJECTION_REASON_ID" INTEGER NULL,
+  "ADDITIONAL_INFO" VARCHAR2(4000) NULL,
+  "EXECUTION_CONDITION" VARCHAR2(4000) NULL,
   "CANCELLATION_CONDITION" VARCHAR2(4000) NULL,
-  "EXPIRES_AT"             TIMESTAMP NULL,
-  "PROPOSED_AT"            TIMESTAMP NULL,
-  "PREPARED_AT"            TIMESTAMP NULL,
-  "EXECUTED_AT"            TIMESTAMP NULL,
-  "REJECTED_AT"            TIMESTAMP NULL
-)
-/
+  "EXPIRES_DTTM" TIMESTAMP NULL,
+  "PROPOSED_DTTM" TIMESTAMP NULL,
+  "PREPARED_DTTM" TIMESTAMP NULL,
+  "EXECUTED_DTTM" TIMESTAMP NULL,
+  "REJECTED_DTTM" TIMESTAMP NULL
+);
 
 CREATE INDEX L_XPK_TRANSFERS ON "L_TRANSFERS"
-  ("TRANSFER_ID" ASC)
-/
-
+  ("TRANSFER_ID" ASC);
 ALTER TABLE "L_TRANSFERS" ADD CONSTRAINT L_PK_TRANSFERS PRIMARY KEY
-  ("TRANSFER_ID")
-/
-
+  ("TRANSFER_ID");
 CREATE INDEX L_XAK_TRANSFERS ON "L_TRANSFERS"
   ("TRANSFER_UUID" ASC);
-/
-
 ALTER TABLE "L_TRANSFERS" ADD CONSTRAINT L_AK_TRANSFERS UNIQUE
   ("TRANSFER_UUID");
-/
-
-ALTER TABLE "L_TRANSFERS" ADD (CONSTRAINT "FK_REJECTION_REASON_ID__TRANSF"
+CREATE BITMAP INDEX L_XIF_TRANSFERS_STATUS ON "L_TRANSFERS"
```

Comment:
You are using a bitmap index because there are few distinct `status_id`s? Is this Abrahams' suggestion?